app/models/rental.rb

```diff
@@ -0,0 +1,42 @@
+class Rental < ApplicationRecord
+  belongs_to :customer
+  belongs_to :video
+
+  validates :video_id, presence: true
+  validates :customer_id, presence: true
+  
+  def set_due_date
+   self.due_date = DateTime.now + 1.week
+  end
+  def set_videos_checked_out_count
+    self.customer.checked_out_video
+    self.videos_checked_out_count = self.customer.videos_checked_out_count
+  end
+  def set_available_inventory
+    self.video.checked_out
+    self.available_inventory = self.video.available_inventory
+  end
```

Comment:
The same could be said here about available_inventory as I said in my comment above about videos_checked_out_count,